### PR TITLE
feat: Installing requirements inside setup.py

### PR DESCRIPTION
**Pull request by jazibhumayun**
To install all requirements/base.in as part of edx-prefectutils package creation and publishing, installing it inside setup.py file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -36,7 +65,6 @@ def get_version(*file_paths):
 
 
 
-requirements = ['Click>=7.0', ]
```
Comment (jazibhumayun) [inline setup.py:39]: this is being installed inside `base.in` but please whoever reviews this PR. Please confirm if the the removal of  line 39 and line 64 `install_requires=requirements` is correct

File: setup.py [hunk 2]
```diff
@@ -23,6 +23,35 @@ def get_version(*file_paths):
...
+def load_requirements(*requirements_paths):
+    """
+    Load all requirements from the specified requirements files.
+    Returns:
+        list: Requirements file relative path strings
+    """
+    requirements = set()
+    for path in requirements_paths:
+        requirements.update(
+            line.split('#')[0].strip() for line in open(path).readlines()
```
Comment (pwnage101) [inline setup.py:35]: Is it necessary to strip comments here if there's already the check below on line 50?

```
         line.startswith('#') or
```

Comment (bmedx) [reply setup.py:35]: I think this is to split out trailing comments after package names like:

`foo==1.0  # This is the foo package`

**Review by bmedx — APPROVED** [review]
This seems ok, I'm not sure where some of the exclusion in `is_requirement` came from but they don't seem harmful.

**Comment by bmedx** [comment]
I think the CI errors are caused by not having `requirements/base.in` in `MANIFEST.in`.

**Review by pwnage101 — APPROVED** [review]

**Comment by jazibhumayun** [comment]
> I think the CI errors are caused by not having `requirements/base.in` in `MANIFEST.in`.

on point, thanks (Y)

**Pull request merged**